README.md

```diff
@@ -9,24 +9,27 @@ _Having problems? want to contribute? join our [community slack](http://devtools
 
 > Generate a CHANGELOG from git metadata
 
-It's recommended you use the high level [standard-version](https://github.com/conventional-changelog/standard-version) library, which handles the entire tagging and CHANGELOG
-generation process.
-
-Alternatively, the [conventional-changelog-cli](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) allows you to generate a CHANGELOG from the
-command line.
-
 ## About this Repo
 
 The conventional-changelog repo is managed as a [monorepo](https://github.com/babel/babel/blob/master/doc/design/monorepo.md); it's composed of many npm packages.
 
 The original `conventional-changelog/conventional-changelog` API repo can be
 found in [packages/conventional-changelog](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog).
 
+## Getting started
+
+It's recommended you use the high level [standard-version](https://github.com/conventional-changelog/standard-version) library, which is a drop-in replacement for npm's `version` command, handling automated version bumping, tagging and CHANGELOG generation.
+
+Alternatively, if you'd like to move towards completely automating your release process as an output from CI/CD, consider using [semantic-release](https://github.com/semantic-release/semantic-release).
```

Comment:
Forgot, can we also reference Lerna for mono-repository project developers?

Reply:
Good call @hutson, I think this would be another good library to call out; @evocateur perhaps you could share a blurb you'd like us to use?